config/scim.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
 return [
-    "publish_routes" => true
+    "publish_routes" => true,
+    'omit_main_schema_in_return' => true
```

Comment:
Maybe it should be `false` by default, seems to be affecting tests.

Reply:
Totally agree. Perhaps tests should be introduced to test this config, but for now I think it's fine.